docs/docs/APIGuide/Layers/Convolution-Layers.md

```diff
@@ -1831,17 +1831,17 @@ output m is : [[[[[ 0.80314147  0.80314147  0.7915861   0.7915861 ]
 ```
 
 
-## LocallyConnected1D ##
+## LocallyConnected2D ##
 **Scala:**
 ```scala
 +val module = LocallyConnected1D(
 +  nInputFrame,inputFrameSize, outputFrameSize, kernelW, strideW = 1, propagateBack = true,
 +  wRegularizer = null, bRegularizer = null, initWeight = null, initBias = null,
 +  initGradWeight = null, initGradBias = null
 +  )
-+```
-+**Python:**
-+```python
+```
+**Python:**
+```python
 +module = LocallyConnected1D(
 +  n_input_frame, input_frame_size, output_frame_size, kernel_w, stride_w = 1, propagate_back = True,
```

Comment:
Need to copy from https://github.com/intel-analytics/BigDL/blob/master/pyspark/bigdl/nn/layer.py#L1310